opentracing-mock/src/main/java/io/opentracing/mock/MockSpan.java

```diff
@@ -262,4 +276,12 @@ static long nextId() {
     static long nowMicros() {
         return System.currentTimeMillis() * 1000;
     }
+
+    private synchronized void finishedCheck(String message) {
+        if (finished) {
+            RuntimeException ex = new IllegalStateException(message);
+            errors.add(ex);
```

Comment:
First, I do not object to it, either. I am just curious about why need the `errors` list? It's like you want to check the list rather than count the `catch exception`?